lib/security/aes.js

```diff
@@ -0,0 +1,40 @@
+var CryptoJS = require("crypto-js");
+
+/**
+* Encrypt an Object using AES with a specified secret key.
+* @param {object} data - the data to encrypt.
+* @param {string} key - the secret key to encrypt with.
+* @returns {string} ciphertext
+*/
+function encrypt(data, key) {
+  if (!data || !key) {
+    return new Error("Cannot Encrypt Profile Data - Data or Secret key is Null.");
```

Comment:
This will throw an exception in the browser though. I just want an error object to be returned to check if the encryption/decryption failed.